tests/Console/E2ETest.php

```diff
@@ -40,7 +40,7 @@ protected function setUp()
         // Without overcommit this test fails with `proc_open(): fork failed - Cannot allocate memory`
         if (strpos(PHP_OS, 'Linux') === 0 &&
             is_readable('/proc/sys/vm/overcommit_memory') &&
-            file_get_contents('/proc/sys/vm/overcommit_memory') == 2) {
+            file_get_contents('/proc/sys/vm/overcommit_memory') === 2) {
```

Comment:
This won't work, you have to cast to int here because `file_get_contents` always returns string.